web-server.js

```diff
@@ -101,6 +114,11 @@ StaticServlet.prototype.handleRequest = function(req, res) {
   if (parts[parts.length-1].charAt(0) === '.')
     return self.sendForbidden_(req, res, path);
   fs.stat(path, function(err, stat) {
+    if (req.url.pathname == '/js/config/local.conf.js' && err) {
+      res.write("window.thServiceDomain = '" + DEFAULT_SERVICE_DOMAIN + "'");;
```

Comment:
There's a double semi-colon at the end of this line. Move it after the quote. ;)

```js
res.write("window.thServiceDomain = '" + DEFAULT_SERVICE_DOMAIN + ";'");
```